src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_model.jl

```diff
@@ -53,24 +53,28 @@ default_free_surface(grid::XYRegularRG; gravitational_acceleration=g_Earth) =
 default_free_surface(grid; gravitational_acceleration=g_Earth) =
     SplitExplicitFreeSurface(grid; cfl = 0.7, gravitational_acceleration)
 
+default_momentum_advection(grid) = CenteredSecondOrder()
+
+default_momentum_advection(::OrthogonalSphericalShellGrid) = VectorInvariant()
```

Comment:
```suggestion
default_momentum_advection(grid) = CenteredSecondOrder()
default_momentum_advection(::OrthogonalSphericalShellGrid) = VectorInvariant()
```

what's your thought on 

```julia
default_momentum_advection(grid) = VectorInvariant()
```

Reply:
It might be advantageous. So that the momentum equation is by default in the vector invariant form for the HydrostaticFreeSurfaceModel

Reply:
This default makes sense to me.

Reply:
nice